src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -148,7 +151,7 @@ const AccordionPanel = forwardRef(
             </Box>
           )}
         </Button>
-        <Box role="region" border={contentBorder}>
+        <Box role="region" border={contentBorder} aria-labelledby={uniqueId}>
```

Comment:
Instead of assigning an id and setting `aria-labelledby` could we pull `aria-label` out of `..rest` and apply it to this Box? If `aria-label` is not defined we can try to fall back to `label` if it is a string.

```
<Box 
  role="region" 
  border={contentBorder} 
  aria-label={ariaLabel || (typeof label === 'string' ? label : undefined)}
>
```

Reply:
I'm following the accordion pattern from [ARIA Authoring Practices Guide (APG)](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/) where they use "aria-labelledby" and link it to the Button.
Their example https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/.

To use aria-label on the Box component we would have to guarantee that it's value be unique otherwise the same violation shows.

Duplicate label show as a violation here: [landmark-unique rule](https://dequeuniversity.com/rules/axe/4.9/landmark-unique?application=axeAPI)
![image](https://github.com/user-attachments/assets/3cf570de-e122-47eb-8348-784fdc53da00)

Reply:
Thanks for the explanation and linking the APG guidance, this is helpful. Chatting with some of the other grommet devs on this, I'll keep you posted...